### ISSUE TITLE
Cannot add new slides

### ISSUE THREAD

**Issue by brenjt** [biati-digital/glightbox#166]
**Describe the bug**
When calling `insertSlide` there is a javascript error `Uncaught TypeError: Cannot read property 'querySelectorAll' of undefined`

**Are you able to reproduce the bug in the demo site**
Yes, here is a demo with the most basic code to see the error: https://codepen.io/brenjt/pen/Exypprj

**Expected behavior**
No errors and the ability to insert more slides

**Post the code you are using**
```
var exampleApi = GLightbox({ selector: null });
exampleApi.insertSlide({
    href: 'https://picsum.photos/1200/800',
});
```

**Desktop:**
All

**Smartphone:**
All

**Comment by wzol** [comment]
Same here. :(

**Comment by biati-digital** [comment]
Fixed in the latest update 3.0.6
